### PR TITLE
Create 2020-02-05-equity-4-who.md

### PR DESCRIPTION
**Pull request by patcon**
Participants gave permission to host these publicly

### CODE REVIEW

File: _posts/informational-interviews/2020-02-05-equity-4-who.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+- ben: fiscal sponsorship overview + breakdown
+    - fiscal sponsorship can mean a few things.
+    - can be very heavy. taking on legal contracts like leases. NOT what we're doing
+    - Hypha dealing in **donations, NOT services**. we're starting with support for no-strings donations, so no taxes need be charged. but using our system for transactions like selling event tickets would be different (taxes).
+    - need records for payouts. claims can be through OC website. needs to submit receipts/invoices. accountant needs them.
+    - meaghan: current = excel spreadsheet.
+        - requested funds. uw email. $38,000 in backlog of requests. working through.
+    - payout ways we currently use: bank transfer, and paypal.
+    - pc: can we do e-transfer? bl: requires more clicking, and our bank is not great.
+        - #update confirmed later than only one member of our co-op (yj) can perform e-transfers, so that's too complicated for us.
```
Comment (patcon) [inline _posts/informational-interviews/2020-02-05-equity-4-who.md:52]: look right @benhylau?

File: _posts/informational-interviews/2020-02-05-equity-4-who.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    - bl: fees. **money in** = 5%. **money out** = no extra charge.
+        - #update bl realized there were still processing charges on payout (which e-transfer may not have), but just minimum of payment platform, e.g. paypal (nothing taken by OC platform or Hypha host)
+        - 5% breakdown
+            - 15% of that 5% goes to our platform (OC)
+            - ~85% of that 5% goes to our accountant
+            - ~10% to us (very low)
```
Comment (patcon) [inline _posts/informational-interviews/2020-02-05-equity-4-who.md:58]: @benhylau did i capture your realization properly (about money out costs)?

@benhylau also, my breakdown math doesn't add up. can you confirm how you explained?

**Comment by patcon** [comment]
Good to merge after ben's input

File: _posts/informational-interviews/2020-02-05-equity-4-who.md (hunk 1 above)
Comment (patcon) [reply _posts/informational-interviews/2020-02-05-equity-4-who.md:52]: Confirmed: https://chat.tomesh.net/#/room/#hyphacoop-private:tomesh.net/$16125411951440nCENb:tomesh.net

File: _posts/informational-interviews/2020-02-05-equity-4-who.md (hunk 2 above)
Comment (benhylau) [reply _posts/informational-interviews/2020-02-05-equity-4-who.md:58]: 85% of that 5% goes to us mainly to pay our accountant

We do not know if we will operate at a loss or gain, our accountant bills by hour not a % of how much money gets processed.

Comment (patcon) [reply _posts/informational-interviews/2020-02-05-equity-4-who.md:58]:
```suggestion
    - bl: fees. **money in** = 5%. **money out** = no extra charge.
        - #update bl realized there were still processing charges on payout (which e-transfer may not have), but just minimum of payment platform, e.g. paypal (nothing taken by OC platform or Hypha host)
        - 5% breakdown
            - 15% of that 5% goes to our platform (OC)
            - ~85% of that 5% goes to us (all to accountant while we're low volume)
```

**Pull request merged**